7-animation/2-css-animations/article.md

```diff
@@ -1,14 +1,14 @@
 # CSS-animations
 
-CSS animations allow to do simple animations without JavaScript at all.
+CSS animations make it possible to do simple animations without JavaScript at all.
 
-JavaScript can be used to control CSS animation and make it even better with a little of code.
+JavaScript can be used to control CSS animations and make them even better, with little code.
 
 ## CSS transitions [#css-transition]
 
 The idea of CSS transitions is simple. We describe a property and how its changes should be animated. When the property changes, the browser paints the animation.
 
-That is: all we need is to change the property. And the fluent transition is made by the browser.
+That is, all we need is to change the property; and, the fluid transition will be done by the browser.
```

Comment:
Is the semicolon needed? `;`

Reply:
The syntax seems just wrong around there.

Reply:
You're right, it isn't needed.

Reply:
> The syntax seems just wrong around there.

It indeed looks a bit awkward.

```suggestion
That is, all we need is to change the property, and the fluid transition will be done by the browser.
```